tests/www/test_views.py

```diff
@@ -496,6 +496,22 @@ def test_last_dagruns(self):
         resp = self.client.get('last_dagruns', follow_redirects=True)
         self.check_content_in_response('example_bash_operator', resp)
 
+    def test_last_dagruns_success_when_selecting_dags(self):
+        resp = self.client.get('last_dagruns?dag_ids=example_subdag_operator', follow_redirects=True)
```

Comment:
Could you add some testing around the "unquoting" of dag_ids?

Reply:
I can do but it is just `encodeURIComponent` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent and urllibs `unquote`. I like we would be unit testing this library here as opposed to airflow

Reply:
Ahh ok in that point I'm good. Once tests pass I will merge.